app/components/utils/mandatee-selector.js

```diff
@@ -2,19 +2,25 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { isEmpty } from '@ember/utils';
 import { tracked } from '@glimmer/tracking';
-import { timeout, task, restartableTask } from 'ember-concurrency';
+import { restartableTask, task, timeout } from 'ember-concurrency';
 import { inject as service } from '@ember/service';
-import { CURRENT_GOVERNMENT_BODY } from 'frontend-kaleidos/config/config';
 
 const VISIBLE_ROLES = [
   'http://themis.vlaanderen.be/id/bestuursfunctie/5fed907ce6670526694a03de', // Minister-president
-  'http://themis.vlaanderen.be/id/bestuursfunctie/5fed907ce6670526694a03e0' // Minister
+  'http://themis.vlaanderen.be/id/bestuursfunctie/5fed907ce6670526694a03e0', // Minister
```

Comment:
This list will not be sufficient when searching for mandatees from before 06/1995. Roles at that time were 'Voorzitter' and 'Gemeenschapsminister'.

Reply:
Small thought about this:

- What about vice-minister-president and vice-voorzitter? Shouldn't they be visible as well?

- Also, if this is not parameterized & basically all roles are currently shown: what's the point of having this filtering?

Reply:
Also, in KAS-3344 you'll see that this component will be upgraded to have an option to show **all** mandatees, including those not in a `besluit:Bestuursorgaan`. So in that case the filtering will have to be bypassed as well.

Reply:
I assume the filters on role are there because a vice-minister can never be assigned to an agendaitem in Kaleidos. They always act in their role of minister.
Just adding the URIs of 'Voorzitter' and 'Gemeenschapsminister' should fix it. And, as you mention, in a next story the users will have an option to bypass the filter.